helm/thingsboard/values.yaml

```diff
@@ -0,0 +1,285 @@
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+http:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-http-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8082
+    protocol: TCP
+    name: http
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8082
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+coap:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-coap-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8083
+    protocol: UDP
+    name: coap
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8083
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+webui:
+  # kind can be either Deployment or StatefulSet
+  kind: Deployment
+  image:
+    repository: thingsboard/tb-web-ui
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8084
+    protocol: TCP
+    name: http
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8084
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+cassandraInitDB:
+  keyspace: thingsboard
+  replication:
+    class: SimpleStrategy
+    factor: 1
+
+cassandra:
+  # Database can be either postgres or hybrid (entities in postgres and timeseries in cassandra)
+  # for an postgresql database set cassandra.enabled as false
+  # for an hybrid database set cassandra.enabled as true
+  enabled: false
+  replicaCount: 1
+  dbUser:
+    password: setplease
+
+postgresql-ha:
+  nameOverride: postgresql-ha
+  postgresql:
+    database: thingsboard
+    replicaCount: 1
+    password: setplease
+    repmgrPassword: setplease
+  postgresqlImage:
+    tag: 12
+  pgpool:
+    adminPassword: setplease
+    replicaCount: 1
+  pgpoolImage:
+    tag: 4
+
+redis:
+  # Set architecture to either standalone or replication
+  architecture: standalone
+  auth:
+    password: setplease
```

Comment:
We better add these sections in here, expect HA issues otherwise:
```
  master:
    kind: Deployment
    disableCommands:
      - FLUSHALL
  replica:
    replicaCount: 0
```

Reply:
add or replace? Do we maintain standalone architecture by default? I'd like to keep a default of minimum resources needed but allowing HA deployments

Reply:
> add or replace:

merge :sunglasses: 
```
redis:
  architecture: standalone
  auth:
    password: setplease
  master:
    kind: Deployment
    disableCommands:
      - FLUSHALL
  replica:
    replicaCount: 0
```
Oh, and, the last time I checked "replication" didn't work well in ThingsBoard, maybe we can remove that comment...